composables/pwa.ts

```diff
@@ -0,0 +1,6 @@
+// https://thomashunter.name/posts/2021-12-11-detecting-if-pwa-twa-is-installed
+
+const ua = navigator.userAgent
+const ios = ua.match(/iPhone|iPad|iPod/)
+const standalone = window.matchMedia('(display-mode: standalone)').matches
+export const isPWAInstalled = !!(standalone || (ios && !ua.match(/Safari/)))
```

Comment:
can we inline this in plugins/pwa.client.ts instead to make it clearer?

Reply:
this is part of the TODO, maybe we can revert that change

Reply:
Looks good to me to merge whenever you're happy. I would do as you've suggested:

> maybe we can keep current code (just remove the comment and undo moving isPWAInstalled to its own module)